addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -19,10 +23,10 @@
         <t t-if="event.event_ticket_ids[-1].price_reduce > 0">
             <span class="text-dark">
                 From
-                <span t-esc="event.event_ticket_ids[0].price_reduce" t-options="{'widget': 'monetary', 'display_currency': website.pricelist_id.currency_id}"/>
+                <span t-esc="event.event_ticket_ids[0].price_reduce" t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.get_current_pricelist().currency_id}"/>
```

Comment:
AFAIK, there is no need to change `display_currency` because `website.pricelist_id == website.get_current_pricelist()` (cf _compute_pricelist_id in website_sale)